src/libopensc/ctx.c

```diff
@@ -135,6 +135,7 @@ static const struct _sc_driver_entry internal_card_drivers[] = {
 #endif
 	{ "masktech",	(void *(*)(void)) sc_get_masktech_driver },
 	{ "esteid2018",	(void *(*)(void)) sc_get_esteid2018_driver },
+	{ "esteid2025",	(void *(*)(void)) sc_get_esteid2025_driver },
```

Comment:
afaik given that the match card does APDU transaction to the card, it should come after the line 144 or later.